src/main_menu.cpp

```diff
@@ -359,8 +359,19 @@ bool main_menu::opening_screen()
     print_menu( w_open, 0, iMenuOffsetX, iMenuOffsetY );
 
     if( !assure_dir_exist( FILENAMES["config_dir"] ) ) {
-        popup( _( "Unable to make config directory. Check permissions." ) );
-        return false;
+        if( query_yn( _( "Unable to make config directory.\nTry to create parents?" ) ) ) {
+            size_t slash_idx = 0;
+            do {
+                slash_idx = FILENAMES["config_dir"].find_first_of( "\\/", slash_idx + 1 );
+                assure_dir_exist( FILENAMES["config_dir"].substr( 0, slash_idx + 1 ) ); 
+            } while ( slash_idx != std::string::npos );
+            if ( !assure_dir_exist( FILENAMES["config_dir"] ) ) {
+                popup( _( "Unable to make config directory. Check permissions." ) );
+                return false;
+            }
+        } else {
+            return false;
```

Comment:
If the user chooses not to make the directory, some sort of response message should still show up before hitting return false so they understand why the config file isn't being made.  
Perhaps something like "The config directory not made due to lack of parent directories"
This way, if they come back to play the game, they will remember to say yes to your query, hopefully.

Reply:
Sure I could add that although it is kind of implied through the first query but it doesn't hurt to be extra clear

Reply:
I removed the query so this change is no longer necessary